app/src/main/java/it/niedermann/owncloud/notes/preferences/PreferencesFragment.java

```diff
@@ -31,12 +29,18 @@
     private BrandedSwitchPreference wifiOnlyPref;
     private BrandedSwitchPreference gridViewPref;
     private BrandedSwitchPreference preventScreenCapturePref;
+    private BrandedSwitchPreference backgroundSyncPref;
 
     @Override
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
     }
 
+    /**
+     * Turn background sync from from {@link String} values to {@link Boolean} values
+     * https://github.com/stefan-niedermann/nextcloud-notes/issues/1168
+     */
```

Comment:
While this description might fit to the Migration, it does not provide any value to this method. Please remove it or merge it with the description in the Migration.